examples/tests/valid/json_static.w

```diff
@@ -1,7 +1,7 @@
 bring cloud;
 
 // Get Keys
-let x = Json {a: 123, b: {c: 456, d: 789}};
+let x = Json {a: 123, b: {c => 456, d => 789}};
```

Comment:
This shouldn't work

Reply:
I have removed map and set from being valid json fields.